src/deepqmc/wf/paulinet/r_backflow.py

```diff
@@ -0,0 +1,38 @@
+import torch
```

Comment:
I'd move this to `deepqmc.extra` actually, since we don't really test this code.